app/models/loader/types.rb

```diff
@@ -199,8 +199,18 @@ def create!
     class User < Record
       def_delegators :@policy_object, :public_keys, :account, :role_kind, :uidnumber, :restricted_to
 
+      def check_user_creation_allowed(res_id:)
```

Comment:
Abbreviations make code harder to read.  Mind changing `res_id` to `resource_id` (assuming that's what you're abbreviating)?

Reply:
done